android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -97,6 +107,9 @@ public void onCreate() {
             Picasso picasso = new Picasso.Builder(this).downloader(new OkHttp3Downloader(httpClient)).build();
             Picasso.setSingletonInstance(picasso);
```

Comment:
Best approach would be to pass picassoInstance made using PicassoCache in the Picasso.setSingletonInstance method. This would make sure every time that instance without changing code everywhere in application.